packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -969,6 +1069,9 @@ class RectangularSliderTrackShape extends SliderTrackShape {
     bool isEnabled,
     bool isDiscrete,
   }) {
+    isEnabled = isEnabled ?? false;
```

Comment:
Can these 2 be set to false as default values in the param list instead of here?

Reply:
Done